app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -3,7 +3,7 @@ def google_oauth2
     auth = request.env["omniauth.auth"]
     if current_user
       unless current_user.has_google?
-        Identity.create(:uid => auth.uid, :provider => auth.provider, :user_id => current_user.id)
+        Identity.create(uid: auth.uid, provider: auth.provider, user_id: current_user.id, image_url: auth.info.image)
```

Comment:
It is not bad approach.
The other option would be to copy image to `avatar` object. In this case, we can grab an original from google and do all standard modifications with `paperclip` (resizing, cropping, etc)